### PR TITLE
[locale.time.get] Replace 'ISO/IEC 9945' with 'POSIX'

### PR DESCRIPTION
**Pull request by jensmaurer**
We introduce POSIX as an alias for ISO/IEC 9945, and we use 'POSIX'
everywhere else.

### CODE REVIEW

**Comment by tkoeppe** [comment]
@jwakely: could you please take a look?

**Review by jwakely — APPROVED** [review]

**Comment by jwakely** [comment]
I missed this in [P2227](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2020/p2227r0.html) but there are no changes to the `strptime` conversion specifications between [POSIX 2001](https://pubs.opengroup.org/onlinepubs/009696799/functions/strptime.html) and [POSIX 2008](https://pubs.opengroup.org/onlinepubs/9699919799/functions/strptime.html) except for clarifications of the intended meaning.

I'm not fond of the term "conversion directive" here though, POSIX says "directives" contain "conversion specifications". There are no conversion directives.

**Review by jwakely — CHANGES_REQUESTED** [review]

File: source/locales.tex [hunk 1]
```diff
@@ -3535,7 +3535,7 @@
 until it has extracted and assigned those \tcode{tm} members, and
 any remaining format characters,
 corresponding to a conversion directive appropriate for
```
[the remark this answers is not in the document]
Comment (jensmaurer) [reply source/locales.tex]: Added this fix in a separate commit. Also found additional mentions nearby; fixed them, too.

**Comment by jensmaurer** [comment]
@jwakely , please have another look.

**Review by jwakely — APPROVED** [review]

**Comment by jensmaurer** [comment]
@tkoeppe , this looks ready.

**Pull request merged**